Libraries/StyleSheet/processColor.js

```diff
@@ -10,33 +10,29 @@
  */
 'use strict';
 
-var tinycolor = require('tinycolor');
 var Platform = require('Platform');
 
+var tinycolor = require('tinycolor');
+
 /* eslint no-bitwise: 0 */
 function processColor(color) {
-  if (!color || typeof color === 'number') {
+  if (color === undefined) {
     return color;
-  } else if (color instanceof Array) {
-    return color.map(processColor);
-  } else {
-    var color = tinycolor(color);
-    if (color.isValid()) {
-      var rgb = color.toRgb();
-      // All bitwise operations happen on 32-bit numbers, so we shift the 1 first
-      // then multiply it with the actual value.
-      var colorInt = Math.round(rgb.a * 255) * (1 << 24) + rgb.r * (1 << 16) + rgb.g * (1 << 8) + rgb.b;
```

Comment:
so, we used to parse a string to an object, just to convert it back to a number. Might as well work on numbers all the way through. Should be faster and it makes the code easier